models2/CommentModel.es6.js

```diff
@@ -2,6 +2,8 @@ import RedditModel from './RedditModel';
 import Record from '../apiBase/Record';
 import { COMMENT, COMMENT_LOAD_MORE } from './thingTypes';
 
+import votable from './mixins/votable';
```

Comment:
How do we feel about this style of adding functionality? We could also consider decorators as a cleaner syntax.